unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PluginSystem/PluginSystemFactory.cs

```diff
@@ -34,6 +34,7 @@ public static PluginSystem Create()
             pluginSystem.Register<UUIDEventsPlugin>(() => new UUIDEventsPlugin());
             pluginSystem.Register<UIComponentsPlugin>(() => new UIComponentsPlugin());
             pluginSystem.Register<CoreComponentsPlugin>(() => new CoreComponentsPlugin());
+            pluginSystem.Register<ECS7ComponentsPlugin>(() => new ECS7ComponentsPlugin());
```

Comment:
just in case, let's not add the plugin to the factory yet :)
since we don't need to do it right now, I would prefer to avoid executing any ecs7 code in prod for the moment